target/arduino/export.bat

```diff
@@ -5,106 +5,101 @@
-::                              user is informed and the script is aborted.
-::  -r  --replace ^<directory^> Replace Arduino library at specified location erasing existing files.
-::                              When a directory is found with equal name the 
-::                              user is requested to confirm deletion of the directory.
 
 :: \name Remko Welling (pe1mew@gmail.com)
 
 SET option=%1
 SET source_directory=%~dp0
-SET target_directory=%2
+
+:: Select operation: "-" is command, else run
+IF  "%option:~0,1%"=="-" (
+   GOTO :CASE
+) ELSE (
+   GOTO :RUN
+)
+GOTO :EOF
+
+:CASE
+:: Command handeling
 
 2>NUL CALL :CASE_%option% 
 
 IF ERRORLEVEL 1 CALL :DEFAULT_CASE
 
 EXIT /B
 
-:CASE_-r
-:CASE_--replace
-:: remove existing folder structure and copy new library at specified location.
-   IF EXIST "%target_directory:"=%\BasicMAC" (
-      GOTO :DO_REMOVE
-   ) ELSE (
-      ECHO [ERROR] Destination directory "%target_directory:"=%\BasicMAC" does not exist. Please correct destination name.
-   )
-GOTO :EOF
-
-:CASE_-c
-:CASE_--create
-:: create the directory and copy files to it, aborting if it already exists.
-   IF EXIST "%target_directory:"=%\BasicMAC" (
-      ECHO [ERROR] Destination directory "%target_directory:"=%\BasicMAC" does exist. Please correct destination name.
-   ) ELSE (
-      GOTO :DO_CREATE
-   )
-GOTO :EOF
-
 :CASE_-h
 :CASE_--help
    ECHO Create an Arduino-compatible library in the given directory, overwriting existing files.
    ECHO -h  --help    This information.
-   ECHO -c  --create  ^<directory^> Create Arduino library at specified location.
-   ECHO -r  --replace ^<directory^> Replace Arduino library at specified location erasing existing files.
 GOTO :EOF
 
 :DEFAULT_CASE
-   ECHO Unknown command or abort. Type: export.bat -h or --help for options.
+   ECHO Unknown command. Type: export.bat -h or --help for options.
+GOTO :EOF
+
+:RUN
+:: Normal operation.
+   IF NOT EXIST "%option:"=%" (
+      ECHO Target directory "%option:"=%" does not exist.
+      GOTO :EOF
+   ) 
+   IF EXIST "%option:"=%\BasicMAC" (
+      GOTO :DO_REMOVE
+   ) ELSE (
+      GOTO :DO_CREATE
+   )
 GOTO :EOF
 
 :DO_REMOVE
-:: remove existing directory, prompt for acknowledge.
-   ECHO Removing directory "%target_directory:"=%\BasicMAC",
+:: Remove existing directory, prompt for acknowledge.
+   ECHO Removing directory "%option:"=%\BasicMAC",
    SET /P AREYOUSURE=Are you sure (Y/[N])?
    IF /I "%AREYOUSURE%" NEQ "Y" GOTO :EOF
    
-   rmdir /s /q "%target_directory:"=%\BasicMAC"
+   rmdir /s /q "%option:"=%\BasicMAC"
 
 :DO_CREATE
-:: Copy all files in to the library in the target directory
-   ECHO Ceating Arduino library in: "%target_directory:"=%"
-   IF NOT EXIST "%target_directory:"=%\BasicMAC" (
-      md "%target_directory:"=%\BasicMAC"
+:: Create directory structure for library in target directory
+   ECHO Ceating Arduino library in: "%option:"=%"
+   IF NOT EXIST "%option:"=%\BasicMAC" (
```

Comment:
This if can be removed, I think, since you just removed the dir if it existed.

Reply:
it is not case sensitive

Reply:
I want to keep this to ensure that the destination directory structure is capeable to have files copied in to.

Reply:
I'm not sure I follow that. My suggestion was to remove the if, but leave the `md` (just run it unconditionally)?

Reply:
You resolved this, but have not replied to the question, nor made any changes to resolve this. Can you clarify?

Reply:
```
You resolved this, but have not replied to the question, nor made any changes to resolve this. Can you clarify?
```
I do not understad what you are referring to. At July 16 I stated that I want to keep the confirmation.
Probably that a commit from my side was not executed nor observed by you. who can tell? I'll try to update again today.

Reply:
You have not replied to my comment from july 16th:

 > I'm not sure I follow that. My suggestion was to remove the if, but leave the md (just run it unconditionally)?

Reply:
You say you want to "keep the confirmation", but what confirmation do you mean exactly? Is the `IF NOT EXIST "%option:"=%\BasicMAC" (` the confirmation you mean? Or do you mean that `      md "%target_directory:"=%\BasicMAC"` is the confirmation? Since I'm proposing to remove the `if`, but keep the `md` (unless changes to the copy command could make the `md` unneeded, that I'd also prefer to remove those, but that's a separate issue).